qa/suites/rados/singleton/all/stretch-mode-5-mons-8-osds.yaml

```diff
@@ -0,0 +1,63 @@
+roles:
+- - mon.a
+  - mon.b
+  - mgr.a
+  - mgr.b
+  - osd.0
+  - osd.1
+  - osd.2
+  - osd.3
+- - mon.c
+  - mon.d
+  - mgr.c
+  - mgr.d
+  - osd.4
+  - osd.5
+  - osd.6
+  - osd.7
+- - mon.e
+- - client.0
+
+openstack:
+  - volumes: # attached to each instance
+      count: 3
+      size: 10 # GB
+overrides:
+  ceph:
+    conf:
+      global:
+        mon election default strategy: 3
+        osd pool default size: 3
+        osd pool default min size: 2
+      mon:
+        debug mon: 30
+tasks:
+- install:
+- ceph:
+    pre-mgr-commands:
+      - sudo ceph config set mgr mgr_pool false --force
+    log-ignorelist:
+      - overall HEALTH_
+      - \(OSDMAP_FLAGS\)
+      - \(OSD_
+      - \(PG_
+      - \(POOL_
+      - \(CACHE_POOL_
+      - \(OBJECT_
+      - \(SLOW_OPS\)
+      - \(REQUEST_SLOW\)
+      - \(TOO_FEW_PGS\)
+      - slow request
+      - \(POOL_APP_NOT_ENABLED\)
+      - overall HEALTH_
+      - \(MGR_DOWN\)
+      - \(MON_DOWN\)
+      - \(PG_AVAILABILITY\)
+      - \(SLOW_OPS\)
```

Comment:
@kamoltat do you think we can eliminate a few of the log-ignorelist? Or should we split it so we have it before and after the conversion ignorelist ?
my concern is that PG_* will be ignored and we will miss important warnings after the conversion happened

Reply:
That's a good point @NitzanMordhai let me remove most of the ones that can affect the integrity of the test.

Reply:
@NitzanMordhai  I had to put in things like `MON_DOWN` and `PG_AVAILABILITY` since my test involves a lot of failover. Honestly, this grep for warning in the cluster log adds more issues than it actually solves. Maybe worth discussing with the team about it.